cost-analyzer/values.yaml

```diff
@@ -63,6 +63,27 @@ global:
       # username: user
       # password: pwd
 
+  # Azure Monitor Managed Service for Prometheus
+  # See https://github.com/MicrosoftDocs/azure-docs/blob/main/articles/azure-monitor/essentials/prometheus-metrics-overview.md for information
+  # and https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/prometheus-remote-write-virtual-machines for more information on setting this up
+  ammsp:
+    enabled: false
+    prometheusServerEndpoint: http://localhost:8081/
+    remoteWriteService: $<AMMSP_METRICS_INGESTION_ENDPOINT>
+    queryEndpoint: $<AMMSP_QUERY_ENDPOINT>
+
+    aadAuthProxy:
+      enabled: false
+      # per https://github.com/Azure/aad-auth-proxy/releases/tag/0.1.0-main-04-10-2024-7067ac84
+      image: mcr.microsoft.com/azuremonitor/auth-proxy/prod/aad-auth-proxy/images/aad-auth-proxy:0.1.0-main-04-10-2024-7067ac84
```

Comment:
```suggestion
      image: $<IMAGE>  # Example: mcr.microsoft.com/azuremonitor/auth-proxy/prod/aad-auth-proxy/images/aad-auth-proxy:0.1.0-main-04-10-2024-7067ac84
```

Yep I'm in agreement with @jessegoodier here. If the user runs into `ErrImagePullBackoff` I think it should be straightforward as to why that would happen. I don't think we need to add extra error checking here.

And yes I think it's preferred to comment out the image name to reduce the surface area for our image vulnerability scanner. Particularly because this is an optional additional feature.